src/commcare_cloud/ansible/roles/couchdb2/tasks/main.yml

```diff
@@ -9,6 +9,26 @@
     mode: 0644
   notify: restart couchdb2
 
+- name: Ensure that the Erlang VM listens on a known minimum port no.
+  become: yes
+  lineinfile:
+    dest: "{{ couchdb_dir }}/etc/vm.args"
+    line: "-kernel inet_dist_listen_min {{ couchdb_cluster_port }}"
```

Comment:
where is this var `couchdb_cluster_port` defined / used?

Reply:
It will be defined in public.yml for the required environments .